libs/product/src/drivers/magento/transforms/bundled-product-transformers.ts

```diff
@@ -35,9 +35,31 @@ function transformMagentoBundledProductItemOption(option: MagentoBundledProductI
 	return {
 		id: option.id.toString(),
 		name: option.label,
-		price: option.price,
+		price: getPrice(option.product),
+		discount: getDiscount(option.product),
 		quantity: option.quantity,
 		is_default: option.is_default,
 		in_stock: option.product.stock_status === MagentoProductStockStatusEnum.InStock
 	}
 }
+
+/**
+ * A function for null checking an object.
+ */
+function getPrice(product: MagentoProduct): number {
+	return product.price_range && 
+		product.price_range.maximum_price && 
+		product.price_range.maximum_price.regular_price && 
+		product.price_range.maximum_price.regular_price.value !== null
+	? product.price_range.maximum_price.regular_price.value : null;
+}
+
+function getDiscount(product: MagentoProduct): DaffProductDiscount {
+	return product.price_range && 
```

Comment:
Can you add `TODO: use optional chaining` comments to these functions? That will make it easier to find this code for refactoring once we upgrade to Angular 9.

Reply:
added